docs/modules/traits/pages/builder.adoc

```diff
@@ -101,3 +101,19 @@ Syntax: [configmap\|secret]:name[/key], where name represents the resource name,
 |===
 
 // End of autogenerated code - DO NOT EDIT! (configuration)
+
+== Node Selectors
+
+With this trait you will also be able to define node selectors for the `builder`` pod when using the `pod`` build strategy.
+Here is a simple example:
+
+.template.yaml
```

Comment:
As traits are very handy with CLI, it would be good to add also the notation required by `kamel`. For example `kamel run xyz -t builder.nodeSelector...`.